lightning/config/config.go

```diff
@@ -222,6 +222,7 @@ func (cfg *Config) LoadFromGlobal(global *GlobalConfig) error {
 	cfg.TiDB.Host = global.TiDB.Host
 	cfg.TiDB.Port = global.TiDB.Port
 	cfg.TiDB.User = global.TiDB.User
+	cfg.TiDB.Psw  = global.TiDB.Psw
```

Comment:
I am more familiar with "Pwd" as an abbreviation, but do we need to abbreviate this at all?

Reply:
I used `Psw` to mimic the field name in the config

Reply:
What did you mean by "mimic the field name in the config"? There is no `Psw` field in the configuration file.

Reply:
The field can be named "Psw" for consistency with old code (or all renamed to "Password"), but the CLI argument should be named `-tidb-password` not `-tidb-psw`.